analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/StaticLocalInitializationSemaphore.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2022 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Microsoft.CodeAnalysis;
+using StyleCop.Analyzers.Lightup;
+
+namespace SonarAnalyzer.SymbolicExecution.Roslyn.OperationProcessors;
+
+internal sealed class StaticLocalInitializationSemaphore : SimpleProcessor<IStaticLocalInitializationSemaphoreOperationWrapper>
+{
+    protected override IStaticLocalInitializationSemaphoreOperationWrapper Convert(IOperation operation) =>
+        IStaticLocalInitializationSemaphoreOperationWrapper.FromOperation(operation);
+
+    protected override ProgramState Process(SymbolicContext context, IStaticLocalInitializationSemaphoreOperationWrapper semaphore) =>
+        context.State.SetSymbolValue(semaphore.Local, null);
```

Comment:
Would be nice to have a method here `Remove(Symbol)` or `Clear(Symbol)`.

Reply:
Noted for the next sprint

Reply:
Remove(Symbol)